Dockerfile

```diff
@@ -12,47 +12,53 @@ ENV PATH /tmp/google-cloud-sdk/bin:$PATH
 
 WORKDIR /tmp
 
-# Buildpack-deps basic requirements
+# Install dependencies
 RUN apk add --update --no-cache \
         ca-certificates \
         curl \
+        python \
         gnupg \
         wget \
         tar \
-        unzip && \
-        apk add --update --no-cache \
+        unzip \
+        py-crcmod \
+        libc6-compat \
+        openssh-client \
         git \
-        bash && \
-        curl --location https://download.docker.com/linux/static/edge/x86_64/docker-$DOCKER_CLIENT_VERSION.tgz --output /tmp/docker-$DOCKER_CLIENT_VERSION.tgz && \
+        bash
+
+# Install Docker
+RUN curl --location https://download.docker.com/linux/static/edge/x86_64/docker-$DOCKER_CLIENT_VERSION.tgz --output /tmp/docker-$DOCKER_CLIENT_VERSION.tgz && \
         tar --extract --gzip --directory /tmp --file /tmp/docker-$DOCKER_CLIENT_VERSION.tgz && \
         mv /tmp/docker/* /usr/bin && \
-        rm -rf /tmp/* && \
-        curl --location https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip --output terraform.zip && \
+        rm -rf /tmp/*
```

Comment:
could we clean-up the workdir like this after every installation? These should be nothing there as we mount an external volume into it.

Reply:
I clear the `/tmp` folder after whole installation